src/load.lisp

```diff
@@ -0,0 +1,335 @@
+;;; -*- mode:lisp;  coding:utf-8 -*-
+
+
+;;;
+;;; LOAD - load & compile Common Lisp file.
+;;;        Simple implementation of the common Lisp function 'load'
+;;;        for JSCL environment.
+
+;;;
+;;; JSCL is free software: you can redistribute it and/or
+;;; modify it under the terms of the GNU General Public License as
+;;; published by the Free Software Foundation, either version 3 of the
+;;; License, or (at your option) any later version.
+;;;
+;;; JSCL is distributed in the hope that it will be useful, but
+;;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;; General Public License for more details.
+;;;
+;;; You should have received a copy of the GNU General Public License
+;;; along with JSCL.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
Nice!  it's much better to have all this code in this specific file.

Reply:
;)